Framework/API/src/MatrixWorkspace.cpp

```diff
@@ -83,6 +84,31 @@ MatrixWorkspace::~MatrixWorkspace() {
   }
 }
 
+void MatrixWorkspace::setIndexTranslator(
```

Comment:
Is there a good reason for having this method? It's going to make the `MatrixWorkspace` interface sensitive to any future changes we might want to make to the `IndexTranslator` constructor.